packages/dx-vue-core/src/sizer.js

```diff
@@ -0,0 +1,97 @@
+const styles = {
```

Comment:
Can we share these styles between react and vue? Maybe use the `dx-styles` package?

Reply:
I am afraid this add a new dependency for customers.
Or you mean only combine this two file to one, without add to rollup?